DEVELOPER.md

```diff
@@ -1,342 +1,125 @@
-# Developer documentation
+# Kuma GUI
 
-Welcome to the developer docs for the Kuma GUI! If you are considering contributing and helping create an awesome user
-experience, this documentation will help you get started and up to speed. It serves as a guide and a reference for the
-technology, libraries, and things used to build and run the app.
+Welcome to the developer docs for the Kuma GUI application.
 
-Consult the Table of Contents below to navigate these docs.
+If you are considering to contribute and help create an awesome user experience, this documentation will help you get started. It serves as a guide and a reference for the technology, libraries, and things used to build and run the app.
 
-## Table of Contents
+## Table of contents
 
-- [Developer documentation](#developer-documentation)
-  - [Table of Contents](#table-of-contents)
-  - [Dependencies](#dependencies)
+- [Prerequisites](#prerequisites)
+  - [Global dependencies](#global-dependencies)
+  - [Installing project dependencies](#installing-project-dependencies)
+- [Development](#development)
+  - [Starting the development server](#starting-the-development-server)
+    - [Disabling anonymous reports in Kuma](#disabling-anonymous-reports-in-kuma)
+  - [Build the application for production](#build-the-application-for-production)
+  - [Run unit tests](#run-unit-tests)
+  - [Run linters](#run-linters)
+- [Project structure & organization](#project-structure--organization)
```

Comment:
You should get rid of TOCs github generates one for you

Reply:
Explicit TOCs are nice because you can use them in editors, too. Alt+Click for me jumps to the respective heading.

Reply:
Yeah but they are always out of date ;)

Reply:
🤷 They can be kept up-to-date and are better to have in the raw source. I’m keeping them.